portfolio/src/main/webapp/index.html

```diff
@@ -28,16 +28,36 @@ <h1>Marco Bosquez</h1>
         <h2>BACKGROUND</h2>
         <hr></hr>
         <div class = backgroundContent>
-          <p>Hi!! My name is Marco and I'm a student at Tecnológico de Monterrey and I'm currently in Mexico City.</p>
-          <p>Ever since I was a kid I enjoyed videogames and this led me to want to learn how to create my own games.</p>
-          <p>After taking extracurricular courses to create 2D games using unity, I realized how powerful and vast this field could be so I started taking online courses 
-              for programming in C# and Python on my own time. This helped me decide to enroll in Computer Science.</p>
-          <p>My goal is to be able to create elaborate creative experiences using technology!</p>
-
-          <button onclick="addFunction();">Greetings</button>
-          <button onclick="getMarcoStats();">Marco</button>
+            <h1 id='messageToTranslate'> 
+                Hi!! My name is Marco and I'm a student at Tecnológico de Monterrey and I currently live in Mexico City.
+                My goal is to be able to create elaborate creative experiences using technology!
+            </h1>
+
+            <select id="language">
+            <option value="en">English</option>
+            <option value="zh">Chinese</option>
+            <option value="es">Spanish</option>
+            <option value="hi">Hindi</option>
+            <option value="ar">Arabic</option>
+            </select>
+
+            <button id="request-translation">Translate</button>
+            
+            <script src="translate.js"></script>
```

Comment:
This should probably be moved up to the top of this file with the `link` tags. It's easy to miss here. If moving it there results in errors because the DOM content hasn't been loaded and elements are missing, look into using the `defer` attribute.